src/services/car/buyNowApplication.ts

```diff
@@ -0,0 +1,28 @@
+import { postData, handleValidationError, Service } from "../../base"
+
+import { WithValidationError } from "../../types/withValidationError"
+import { BuyNowApplication } from "../../types/models/applications"
+
+export const sendBuyNowApplication = async (
+  listingId: number,
+  buyNowApplication: BuyNowApplication,
+  { recaptchaToken = null, validateOnly = false } = {}
+): Promise<WithValidationError<BuyNowApplication>> => {
+  const path = `/listings/${listingId}/buy-now-applications${
+    validateOnly ? "/validate" : ""
+  }`
+
+  const headers = {
+    ...(recaptchaToken ? { "Recaptcha-Token": recaptchaToken } : {}),
+  }
```

Comment:
do we repeat this line in every API call that has a recaptcha token? would it be worth it pushing it one level down?